website/cloudbuild.yaml

```diff
@@ -0,0 +1,18 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+steps:
+- name: "gcr.io/cloud-builders/gcloud"
+  args: ["app", "deploy"]
```

Comment:
Thanks for commenting!

1. Based on what I've researched on this, the main change that needs to occur is reconfiguring and adding the domain within App Engine. Specifically, [here](https://pantheon.corp.google.com/appengine/settings/domains/add?project=stackdriver-sandbox-230822&folder&organizationId=433637338589). This is something that an owner of the stackdriver-sandbox.dev URL may need to do, since it must be verified.

2. I will be sure to add documentation on testing and deploying site changes! What changes do you think have to occur on the release script?

Reply:
Our release is now automated, so we need to deploy website changes from master branch to AppEngine as part of the release script

Reply:
Instead of directly making it part of the `make-release.sh` script, it would be ideal if we could integrate it into [this GitHub Action](https://github.com/GoogleCloudPlatform/stackdriver-sandbox/blob/master/.github/workflows/push-tags.yml), so we can watch it progress with the other release automations